tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+
+import pytest
+import toml
+
+# Constants used in tests
+PULP_FIXTURES_URL = os.environ.get("PULP_FIXTURES_URL", "https://fixtures.pulpproject.org")
+
+ENV_CONSTANTS = {
+    "PULP_FIXTURES_URL": PULP_FIXTURES_URL,
+    "FILE_REMOTE_URL": PULP_FIXTURES_URL + "/file/PULP_MANIFEST",
+    "CONTAINER_REMOTE_URL": "https://registry-1.docker.io",
+    "CONTAINER_IMAGE": "hello-world",
+    "RPM_REMOTE_URL": PULP_FIXTURES_URL + "/rpm-unsigned",
+    "ANSIBLE_COLLECTION_REMOTE_URL": "https://galaxy.ansible.com/",
+    "ANSIBLE_ROLE_REMOTE_URL": "https://galaxy.ansible.com/api/v1/roles/?namespace__name=elastic",
+}
+
+
+@pytest.fixture
+def cli_settings(tmp_path, monkeypatch):
+    settings = toml.load("tests/settings.toml")
+    settings_path = tmp_path / "config" / "pulp"
+    settings_path.mkdir(parents=True)
+    with open(settings_path / "settings.toml", "w") as settings_file:
+        toml.dump(settings, settings_file)
+    monkeypatch.setenv("XDG_CONFIG_HOME", str(tmp_path / "config"))
+    monkeypatch.setenv("PULP_BASE_URL", settings["cli"]["base_url"])
+    monkeypatch.setenv("VERIFY_SSL", json.dumps(settings["cli"].get("verify_ssl", True)))
```

Comment:
Why are you calling `json.dumps` here?

Reply:
Because that yields lowercase true/false. `True` would break the scripts.
Maybe tolower would be better than trying to mimic toml with json. :nerd_face: